gammapy/background/models.py

```diff
@@ -650,3 +651,26 @@ def compute_rate(self):
         bg_rate = bg_rate.to('MeV-1 sr-1 s-1')
 
         self.bg_rate.data = bg_rate
+
+    def smooth(self):
+        """Smooth the bkg rate with a gaussian 1D kernel.
+
+        """
+        for binE, E in enumerate(self.counts.energy[:-1]):
+            counts = self.counts.data[binE, :]
+            Nev = np.sum(counts).value
+            if (Nev > 0):
+                Np = len(counts)
+                # Number of pixels per sigma of the kernel gaussian to have more than 150 events/sigma
+                Npix_sigma = (150 / Nev) * Np
```

Comment:
Minor: This and the following code could be reduced by using e.g. `np.where()` or `np.clip()`